hack/kube-api-linter.yaml

```diff
@@ -0,0 +1,50 @@
+run:
```

Comment:
Let's configure OWNERS to label `kind/api-change` to diffs for _this_ file, we have this for actual API changes already to auto-assign api approvers and tag the PR:
https://go.k8s.io/owners#filters

I think we can leave the scripts etc to the rest of hack/owners but I'd want to flag API approvers when we add lints

Reply:
I think I've made the correct change to the owners file, if you could verify please

Reply:
Hmm, I don't think that quite works, also I realized this is going to block e.g. fixing the `go:` version on api-approvers

Maybe we use a template and make api-approvers the reviewers/approvers for the source of truth of which lints are enabled here, but not the rest of the golangci config etc. ...

cc @pohly

Reply:
You could put content that you want to own exclusively into a subdirectory as suggested elsewhere, then include it like this:
https://github.com/kubernetes/kubernetes/blob/627ce4946ccd6b67bc18f5a4dda92cf6c583e9b7/hack/golangci.yaml.in#L176-L177

See `hack/update-golangci-lint-config.sh` and `hack/verify-golangci-lint-config.sh`.

Reply:
Doesn't need to be text, as in this example. It can also be YAML.